archive/tar.go

```diff
@@ -419,6 +419,21 @@ func createTarFile(ctx context.Context, path, extractDir string, hdr *tar.Header
 		}
 	}
 
+	// in https://github.com/moby/moby/blob/master/pkg/archive/archive.go, moby still use 'Xattrs'
+	// to deal with file extended attributes, maybe other image build tools also do this way.
+	// Add resolve file attributes from 'hdr.Xattrs' make these images work good
+	for key, value := range hdr.Xattrs {
```

Comment:
I think we can close this PR. As @dmcgowan mentions, the `hdr.Xattrs` has been deprecated.

In > go1.10 version, I read the code in golang code base. The `PAXRecords` will contains all values in `hdr.Xattrs`. please check this https://github.com/golang/go/blob/50bd1c4d4eb4fac8ddeb5f063c099daccfb71b26/src/archive/tar/reader.go#L249

If I understand correctly, after go1.10, user can retrieve the `SCHILY.xattr.` value from `PAXRecords`. `PAXRecords` can provides unify entrypoint to user.